src/back/wgsl/writer.rs

```diff
@@ -1343,6 +1512,16 @@ fn sampling_str(sampling: Sampling) -> &'static str {
     }
 }
 
+fn storage_class_str(storage_class: StorageClass) -> Option<&'static str> {
+    match storage_class {
+        StorageClass::Private => Some("private"),
+        StorageClass::Uniform => Some("uniform"),
+        StorageClass::Storage => Some("storage"),
+        StorageClass::PushConstant => Some("push_constant"),
+        _ => None,
```

Comment:
`WorkGroup` should also be `Some()`. That leaves only `Function` and `Handle` with `None`, so let's list them explictly